modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -102,6 +102,6 @@
 	transition: padding-right 0.5s ease;
 }
 
-.sidebar {
+.content-dashboard.sidebar {
 	transition: transform 0.5s ease;
```

Comment:
Don't know why I did it in this way but I guess this can go inside the previous `.content-dashboard.sidebar`

Reply:
See https://github.com/liferay-frontend/liferay-portal/pull/314/commits/14d4c556cbdf60681ff3cd61ec763446012bf510